core/gui/src/app/dashboard/component/user/user-dataset/user-dataset-explorer/user-dataset-explorer.component.ts

```diff
@@ -213,6 +213,39 @@ export class UserDatasetExplorerComponent implements OnInit {
     }
   }
 
+  onClickDownloadVersionAsZip() {
+    if (this.did && this.selectedVersion && this.selectedVersion.dvid) {
+      this.datasetService
+        .retrieveDatasetVersionZip(this.currentDisplayedFileName)
```

Comment:
retrieve a version zip but pass the entire file path into it is weird.

Reply:
so can you add a function to filter out only `/ownerEmail/datasetName/versionName`?